### DISCUSSION TITLE
Can only use the Companion App now

### DISCUSSION THREAD

**Discussion by rob-jensen-93** [aclap-dev/video-downloadhelper#130] [Help and Q&A]
### Your question

So I was just upgraded to v8.2 and I've come to find out that Downloadhelper no longer asks me if I want downloads to be from the Companion App or not. I've checked my settings, and it says Ask on that prompt, but everything from multiple sites is being automatically done via the Companion App..... 

It may not seem like the hugest deal, but sometimes I would try and download multiple Standard Definition things at once, and it convenient to be able to use BOTH the Companion App, and the "regular" browser download, so that way each thing was being put in a different location, thus eliminating the need to "sort" them, and I could just put everything in a folder just like that.

I don't care if the Companion App is the default, but I've always had a choice as to what I could do with the app, and I'd like to keep having that choice.

### Addon & CoApp version and details

I use firefox as my browser.

**Comment by GrampaWildWilly** [comment]
It is possible that some of your settings got blasted.  You need to go into the VDH settings & make sure things are the way they should be.  It sounds like you need to revisit Behavior -> Download processor.  But look at the rest of your settings as well.  Also, you may discover that your default action has not been kept.  Just go through setting your default action again & you should be fine from that point forward.

**Comment by paulrouget** [comment]
We were not respecting the "Ask" option.

This will be fixed in the upcoming update!
